apps/sequencer/sequencer_config.json

```diff
@@ -13,28 +13,66 @@
     "ETH1": {
       "private_key_path": "/tmp/priv_key_test",
       "url": "http://127.0.0.1:8545",
-      "transaction_retries_count_limit": 5,
```

Comment:
Nit: At some point during PR reviews and discussions I think we decided to use `transaction_retries_count_limit` instead of `transaction_retries_count_before_give_up`. I think we might want to be consistent.